apps/client/src/app/qr/page.tsx

```diff
@@ -1,18 +1,36 @@
-'use-client'
-
+'use client'
 import QRCode from "react-qr-code";
 
+
+function copyText(entryId: string) {
+    navigator.clipboard.writeText(entryId);
+    <p className='mt-8'>Text copied</p>
```

Comment:
This looks strange. Later we will add some notification system. I think we need to remove `<p className='mt-8'>Text copied</p>` for now.

Reply:
Thank you for the comment. Yes, I tried to do notification